locales/en.json

```diff
@@ -401,12 +401,13 @@
     },
     "notifications_settings": "Notifications",
     "preferences": {
+      "continuous_threads": "Continuous Threads",
       "enable_autoplay": "Enable Autoplay",
       "enable_pinch_to_zoom": "Enable pinch to zoom",
       "github_cards": "GitHub Cards",
       "grayscale_mode": "Grayscale mode",
       "hide_account_hover_card": "Hide account hover card",
-      "hide_alt_indi_on_posts": "Hide alt indicator on posts",
+      "hide_alt_indicator_on_posts": "Hide alt indicator on posts",
```

Comment:
can you also update rest of `json` files?

Reply:
Sure, in a couple hours. Any other feedback?

Reply:
it was already there

Reply:
This is just a font issue. I'd like to avoid *more* magic numbers

Reply:
I mean, with the avatar in the publish widget :sweat_smile:

Reply:
Oh yeah, well that's not my fault ;)